provisioning/containers/utils/content-uploader/entrypoint.sh

```diff
@@ -20,4 +20,4 @@ zip "/tmp/$FILENAME.zip" -r "$SOURCE_PATH"
 
 
 echo "Uploading archive..."
-curl -v -X POST -F "binfile=@\"/tmp/$FILENAME.zip\"" -F "filename=$FILENAME.zip" "$DESTINATION_URL"
+curl -v --fail-with-body --request POST --form "binfile=@\"/tmp/$FILENAME.zip\"" --form "filename=$FILENAME.zip" "$DESTINATION_URL"
```

Comment:
`fail-with-body` seems a non-existent option to me

Reply:
Obviously I saw it [here](https://curl.se/docs/manpage.html#--fail-with-body) but I haven't checked if it actually worked